config/routes.rb

```diff
@@ -2,5 +2,10 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
-  # root "articles#index"
+  root "users#index"
+
+  get "/users", to: "users#index"
+  get "/users/:id", to: "users#show"
+  get "/users/:id/posts", to: "posts#index"
+  get "/users/:id/posts/:id", to: "posts#show"
```

Comment:
### **Optional**

- [ ] You can improve your routes by using the resources route

# config/routes.rb

 ```ruby
resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

Reply:
well resolved! ✔️